lib/paperclip/storage/s3.rb

```diff
@@ -153,7 +153,7 @@ def sanitize_hash(hash)
             Proc.new do |style, attachment|
               permission  = (@s3_permissions[style.to_s.to_sym] || @s3_permissions[:default])
               permission  = permission.call(attachment, style) if permission.respond_to?(:call)
-              (permission == DEFAULT_PERMISSIONpublic_read) ? 'http'.freeze : 'https'.freeze
+              (permission == DEFAULT_PERMISSION) ? 'http'.freeze : 'https'.freeze
```

Comment:
Line is too long. [81/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.